### ISSUE TITLE
Nvda does not read alerts in chrome

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#7257]
When opening a few pages in the chrome, the nvda reads alert, and sometimes announces the same word alert several times.
I would like that besides informing alert, the nvda was able to inform which alert.
I do not know visually which alert is displayed.
I opened a few pages in firefox, and the message I get is about blocking flash execution, maybe the alert displayed in chrome is for a reason.

### Steps to reproduce:
Open the page:
Https://www.tecmundo.com.br/wwdc-2017/117390-vivo-acompanhe-wwdc-2017-conheca-ios-11-novos-macbooks.htm

You should hear one or even more than one warning alert.
The expected one would be for nvda to pronounce the alert word and then the alert, for example, the execution of the blocked flash.

### System configuration:
NVDA version:
master 14090,c80e529f

NVDA Installed or portable:
Installed 

Windows version:
> windows 7 64bits

Name and version of other software in use when reproducing the issue:
chrome 58 64bits

### Other questions:

Does the issue still occur after restarting your PC?
yes

Have you tried any other versions of NVDA?
no

**Comment by fernando-jose-silva** [comment]
Follows the log that bypassing opened the call.

Nvda does not read the alert warning even though the add-ons are disabled.

IO - inputCore.InputManager.executeGesture (16:40:25.046):
Input: kb(laptop):enter
IO - tones.beep (16:40:25.812):
Beep at pitch 409.2, for 40 ms, left volume 52, right volume 32
IO - speech.speak (16:40:25.861):
Speaking [u'Ao vivo: acompanhe a WWDC 2017 e conhe\xe7a o iOS 11, novos MacBooks e mais - TecMundo  documento']
IO - speech.speak (16:40:25.869):
Speaking [u'marca b\xe2ner ', u'edi\xe7\xe3o  entrada inv\xe1lida  exigido', u'BUSCAR']
IO - speech.speak (16:40:30.252):
Speaking [u'alerta']
IO - speech.speak (16:40:30.598):
Speaking [u'alerta']
IO - speech.speak (16:40:30.759):
Speaking [u'alerta']
IO - speech.speak (16:40:30.859):
Speaking [u'alerta']
IO - speech.speak (16:40:30.970):
Speaking [u'alerta']
IO - speech.speak (16:40:31.453):
Speaking [u'alerta']
DEBUGWARNING - comtypesMonkeyPatches.newCpbDel (16:40:31.463):
COM pointer <POINTER(IAccessibleText) ptr=0x8ece49c at adff210> already deleted
DEBUGWARNING - comtypesMonkeyPatches.newCpbDel (16:40:31.463):
COM pointer <POINTER(IAccessibleAction) ptr=0x8ece454 at adff350> already deleted
DEBUGWARNING - comtypesMonkeyPatches.newCpbDel (16:40:31.464):
COM pointer <POINTER(IAccessible2) ptr=0x8ecd86c at adff800> already deleted
IO - speech.speak (16:40:31.607):
Speaking [u'alerta']
DEBUGWARNING - comtypesMonkeyPatches.newCpbDel (16:40:31.611):
COM pointer <POINTER(IAccessibleText) ptr=0x8ecc424 at adff7b0> already deleted
DEBUGWARNING - comtypesMonkeyPatches.newCpbDel (16:40:31.611):
COM pointer <POINTER(IAccessibleAction) ptr=0x8ecc3dc at adff300> already deleted
DEBUGWARNING - comtypesMonkeyPatches.newCpbDel (16:40:31.611):
COM pointer <POINTER(IAccessible2) ptr=0x8ecc0c4 at adff530> already deleted
IO - speech.speak (16:40:31.724):
Speaking [u'alerta']
DEBUGWARNING - comtypesMonkeyPatches.newCpbDel (16:40:31.729):
COM pointer <POINTER(IAccessibleText) ptr=0x8ecc034 at adff120> already deleted
DEBUGWARNING - comtypesMonkeyPatches.newCpbDel (16:40:31.729):
COM pointer <POINTER(IAccessibleAction) ptr=0x8ecbfec at adff260> already deleted
DEBUGWARNING - comtypesMonkeyPatches.newCpbDel (16:40:31.729):
COM pointer <POINTER(IAccessible2) ptr=0x8ecbcd4 at adff1c0> already deleted
IO - speech.speak (16:40:31.818):
Speaking [u'alerta']
IO - speech.speak (16:40:32.076):
Speaking [u'alerta']
IO - inputCore.InputManager.executeGesture (16:40:36.418):
Input: kb(laptop):NVDA+f1
INFO - globalCommands.GlobalCommands.script_navigatorObject_devInfo (16:40:36.592):
Developer info for navigator object:
name: u'Ao vivo: acompanhe a WWDC 2017 e conhe\xe7a o iOS 11, novos MacBooks e mais - TecMundo'
role: ROLE_DOCUMENT
states: STATE_READONLY, STATE_FOCUSABLE, STATE_FOCUSED
isFocusable: True
hasFocus: True
Python object: <NVDAObjects.IAccessible.chromium.Document object at 0x0ADAAA10>
Python class mro: (<class 'NVDAObjects.IAccessible.chromium.Document'>, <class 'NVDAObjects.IAccessible.ia2Web.Document'>, <class 'NVDAObjects.IAccessible.ia2Web.Ia2Web'>, <class 'NVDAObjects.IAccessible.IAccessible'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <type 'object'>)
description: None
location: None
value: None
appModule: <'appModuleHandler' (appName u'chrome', process ID 6476) at address 6327410>
appModule.productName: u'Google Chrome'
appModule.productVersion: u'58.0.3029.110'
TextInfo: <class 'NVDAObjects.IAccessible.IA2TextTextInfo'>
windowHandle: 1771590L
windowClassName: u'Chrome_RenderWidgetHostHWND'
windowControlID: 324288704
windowStyle: 1445986304
windowThreadID: 3328
windowText: u'Chrome Legacy Window'
displayText: exception: 'NoneType' object is not iterable
IAccessibleObject: <POINTER(IAccessible2) ptr=0x8eb98c4 at ade2f80>
IAccessibleChildID: 0
IAccessible event parameters: windowHandle=1771590, objectID=-4, childID=-220277
IAccessible accName: u'Ao vivo: acompanhe a WWDC 2017 e conhe\xe7a o iOS 11, novos MacBooks e mais - TecMundo'
IAccessible accRole: ROLE_SYSTEM_DOCUMENT
IAccessible accState: STATE_SYSTEM_READONLY, STATE_SYSTEM_FOCUSED, STATE_SYSTEM_FOCUSABLE, STATE_SYSTEM_VALID (1048644)
IAccessible accDescription: None
IAccessible accValue: u'https://www.tecmundo.com.br/wwdc-2017/117390-vivo-acompanhe-wwdc-2017-conheca-ios-11-novos-macbooks.htm'
IAccessible2 windowHandle: 1771590
IAccessible2 uniqueID: -220277
IAccessible2 role: ROLE_SYSTEM_DOCUMENT
IAccessible2 states: IA2_STATE_OPAQUE (1024)
IAccessible2 attributes: u'tag:#document;explicit-name:true;'

**Comment by pbsinnett** [comment]
It appears that NVDA is saying alert because of page warnings and errors. Here is some text from the browser console.

www-embed-player.js:357 [Violation] 'setTimeout' handler took 150ms
common.bundle.037f55…e90738c195e946.js:4 [Violation] 'setTimeout' handler took 68ms
common.bundle.037f55…e90738c195e946.js:4 [Violation] 'setTimeout' handler took 93ms
117390-vivo-acompanh…ovos-macbooks.htm:1 The resource https://disqus.com/next/config.js was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it Please make sure it has an appropriate `as` value and it is preloaded intentionally.
117390-vivo-acompanh…ovos-macbooks.htm:1 The resource https://c.disquscdn.com/next/embed/common.bundle.037f55c….js was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it Please make sure it has an appropriate `as` value and it is preloaded intentionally.
117390-vivo-acompanh…ovos-macbooks.htm:1 The resource https://c.disquscdn.com/next/embed/lounge.bundle.2fd6d20….js was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it Please make sure it has an appropriate `as` value and it is preloaded intentionally.
117390-vivo-acompanh…ovos-macbooks.htm:1 The resource https://c.disquscdn.com/next/embed/styles/lounge.188f59a….css was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it Please make sure it has an appropriate `as` value and it is preloaded intentionally.
Service Worker termination by a timeout timer was canceled because DevTools is attached.

I've noticed this behavior on other pages with Disqus. It can become quite annoying. I'm not sure if the issue is with NVDA or Chrome, though.

**Comment by Adriani90** [comment]
I think I can reproduce this issue in NVDA 2019.1.1 and Chrome 73.
1. Open the link in the description, not ethat this issue might be reproducible only if you open it for the first time. A dialog wil appear.

Actual: You will hear "alert: Allow button, Block button". The alert itself cannot be read with object navigation or with tab and shift+tab. The message of the alert is even not reported when presing nvda+b to read the whole window. Only OCR with the navigator on the right location will display the message of the alert.


Expected: when the dialog opens, NVDA should report "alert: "message of the allert", allow button block button". The user should be able to review the message of the alert eiterh with object navigation and object review or by pressing shift+tab.

**Comment by Adriani90** [comment]
This example does not work anymore, the dialog with the allert does not appear on Chrome 79 with NVDA 2019.3 Beta 1. @fernando-jose-silva do you have another test case? Are you able to reproduce it with NVDA 2019.3 Beta 1?

**Comment by fernando-jose-silva** [comment]
Many things have changed since the opening of this calling, including the death of the computer used at the time.
Using a new machine with windows 10 1909 build 10.0.18363.476
nvda alfa 18460
chrome chrome
78.0.3904.108
I opened several websites and could not reproduce the problem anymore.
Thank you.

**Comment by Adriani90** [comment]
Thanks for the reply. Closing as works for me.
